android/app/src/main/res/values/connect.sample

```diff
@@ -1,11 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<resources>
```

Comment:
Actually, you don't want to delete connect.sample from the repo. In that case, people outside of Berkeley won't know the format, and won't be able to register their own keys. This is also the reason for the build breakage detected by the continuous build and test system.